Armadillo/src/main/java/com/scribd/armadillo/ArmadilloPlayerChoreographer.kt

```diff
@@ -68,6 +69,14 @@ interface ArmadilloPlayer {
      */
     fun beginPlayback(audioPlayable: AudioPlayable, config: ArmadilloConfiguration = ArmadilloConfiguration())
 
+    /**
+     * Provide new media request data to the currently playing content.
+     * It is an error to call this with a different URL from the currently playing media.
+     * It is also an error to call this when no content is currently loaded.
```

Comment:
Not sure if this will error since it's wrapped in `doIfPlaybackReady` but I could be wrong.

Reply:
`doIfPlaybackReady` generates an error if content is not loaded.